atom/browser/ui/tray_icon_cocoa.mm

```diff
@@ -25,7 +26,13 @@ - (id)initWithIcon:(atom::TrayIconCocoa*)icon {
 }
 
 - (void)handleClick:(id)sender {
-  trayIcon_->NotifyClicked();
+  // Get the position of the frame of the NSStatusItem.
+  NSPoint pos = ([[[NSApp currentEvent] window] frame]).origin;
+  // Flip coordinates to gfx (0,0 in top-left corner) using primary screen.
+  NSScreen* screen = [[NSScreen screens] objectAtIndex:0];
+  pos.y = NSMaxY([screen frame]) - pos.y;
+  gfx::Point position(pos.x, pos.y);
+  trayIcon_->NotifyClicked(position);
```

Comment:
Having both `pos` and `position` in same scope looks a little ugly, maybe just `NotifyClicked(gfx::Point(...))`.